tests/integration/standard/test_connection.py

```diff
@@ -398,3 +398,49 @@ def setUp(self):
             raise unittest.SkipTest(
                 'libev does not appear to be installed properly')
         ConnectionTests.setUp(self)
+
+    def test_watchers_are_finished(self):
```

Comment:
We run ConnectionTests with Libev and Asyncore, not sure why we don't do it with other event loop managers but might be worth to add them